Azure-Samples/ms-identity-javascript-react-tutorial
# Updated package.json and package-lock.json files.

## Purpose
This pull request is meant to address some of the issues caught by `npm audit --production` in some of the samples of this repository.
* ...

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[x] Other... Please describe: Package updates
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
* Run samples in accordance with the `README`. There should be no changes.
```

## What to Check
Verify that the following are valid
* Code sample runs like before
* Running `npm audit --production` shows `0 vulnerabilities`.

## Other Information
* This update is meant to address the issues caught from running audits of production builds, not developer builds.
